### PR TITLE
Add Naikari

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/naikari/naikari/pull/6
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://github.com/naikari/naikari/pull/6

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build io.github.naikari.Naikari

**Comment by Mailaender** [comment]
bot, build io.github.naikari.Naikari

File: io.github.naikari.Naikari.yaml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    make-install-args:
+      - PREFIX=${FLATPAK_DEST}
+    cleanup:
+      - /bin
+      - /share
+    sources:
+      - type: git
+        url: https://github.com/LuaJIT/LuaJIT.git
+        branch: v2.1
+        commit: 4c2441c16ce3c4e312aaefecc6d40c4fe21de97c
```
Comment (hfiguiere) [inline io.github.naikari.Naikari.yaml:27]: you never put a branch.

```suggestion
        # branch: v2.1
        commit: 4c2441c16ce3c4e312aaefecc6d40c4fe21de97c
```

(I leave as a comment for information)

**Review by hfiguiere — CHANGES_REQUESTED** [review]
You probably need cleanup rules for all these `include`

**Comment by hfiguiere** [comment]
There is also a build warning:
```
WARNING: Icon not matching app id in /srv/buildbot/worker/build-x86_64-2/build/builddir/export/share/applications/io.github.naikari.Naikari.desktop: naikari
WARNING: Icon referenced in desktop file but not exported: naikari
```

**Comment by Mailaender** [comment]
bot, build io.github.naikari.Naikari

**Comment by Mailaender** [comment]
bot, build io.github.naikari.Naikari

**Pull request closed without merging**